public/app/listeners/installAppUpgrade.js

```diff
@@ -0,0 +1,28 @@
+const { autoUpdater } = require('electron-updater');
+const logger = require('../logger');
+const { INSTALL_APP_UPGRADE_CHANNEL } = require('../config/channels');
+const { ERROR_GENERAL } = require('../config/errors');
+
+const installAppUpgrade = mainWindow => async (event, shouldUpgrade) => {
+  try {
+    if (shouldUpgrade) {
+      autoUpdater.once('update-downloaded', () => {
+        const msg = 'Update downloaded';
```

Comment:
Can we translate this?

Reply:
Is it necessary since it is displayed only in the logger? Plus I am not sure it is easy to do since it is in a listener (probably we will need to pass the translated text from the front).